### PR TITLE
Fix coverage report workflow

### PR DESCRIPTION
**Pull request by cristianoc72**
Now it's fixed.

### CODE REVIEW

File: clover.xml [hunk 1]
```diff
@@ -0,0 +1,468 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gossi) [inline clover.xml:1]: I think, you can add this file to `.gitignore`

**Review by gossi — COMMENTED** [review]
nice using scrutinizer as dep. I think, we can use one more gitignore addition and then are good?

**Comment by cristianoc72** [comment]
> nice using scrutinizer as dep.

to be honest I don't like to add tools as dependencies at all. But it seems the only way to run ocular with PHP8, by now.
https://github.com/scrutinizer-ci/ocular/issues/54

File: clover.xml (hunk 1 above)
Comment (cristianoc72) [reply clover.xml]: Done. Sorry I forgot it!

**Review by gossi — APPROVED** [review]

**Pull request merged**